staging/gatekeeper/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: "3.4.0-rc.1"
 description: Gatekeeper - Policy Controller for Kubernetes
 name: gatekeeper
-version: 0.6.9
+version: 0.6.10
```

Comment:
just a heads up, we switched to the upstream chart in k-apps for 2.2 https://github.com/mesosphere/kommander-applications/blob/main/services/gatekeeper/3.7.0/release/release.yaml#L13

Reply:
Oh thanks, I totally missed that. I guess that makes the gatekeeper update in this PR useless but w/e.

Reply:
shouldn’t hurt anything!